coda/coda_mdstore/presentation.py

```diff
@@ -32,20 +26,20 @@ def getFileList(url):
     """
 
     fileList = []
-    handle = urllib2.urlopen(url)
+    handle = urllib.request.urlopen(url)
     soup = BSoup(handle)
-    trList = soup.findAll('tr')
+    trList = soup.find_all('tr')
     for tr in trList:
-        tds = tr.findAll('td')
+        tds = tr.find_all('td')
         for td in tds:
-            anchors = td.findAll('a')
+            anchors = td.find_all('a')
             for anchor in anchors:
                 try:
                     # if anchor.contents and "Parent Directory" in contents:
                     if anchor['href'][-1] == "/":
                         continue
                     fileList.append(anchor['href'])
-                except Exception, e:
+                except Exception as e:
                     raise e
```

Comment:
It's the way the code has been written in python2. I think I haven't noticed it before. I can catch a specific Exception here.

Reply:
My question isn't about a specific exception being raised, it is about the act of catching an exception only to then raise it as is. It seems like that is the same thing that would happen if you don't catch the exception at all; the exception would raise the same.

Reply:
Maybe the intention here was to write except clause just for syntax purpose and raised the exception which will happen anyway, instead of doing nothing .
Do you think we can change it to 
```
                try:
                    # if anchor.contents and "Parent Directory" in contents:
                    if anchor['href'][-1] == "/":
                        continue
                    fileList.append(anchor['href'])
                except specific_exception:
                    pass
```

Reply:
Theoretically this could throw a `KeyError` or an `IndexError`, but based on the context of the pages we are using this for, if those occur, there is something unexpectedly wrong with the page, so it might make sense to keep raising an exception. If you aren't comfortable with removing the try/except lines, you can leave it as is.

Reply:
I'd prefer removing try/except clauses from here.